src/Blazor.Auth0.Shared/Blazor.Auth0.Shared.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
The official recommendation is to target the lowest version of .NET Standard possible, and there's no known reason for upgrading it, please revert this change.